docs/dev/setup.rst

```diff
@@ -27,13 +27,11 @@ following dependencies/tools on your machine:
 2. `MySQL Database Server 8 <https://dev.mysql.com/downloads/mysql>`__:
    Artemis uses Hibernate to store entities in a MySQL database.
    Download and install the MySQL Community Server (8.0.x) and configure
-   the ‘root’ user with an empty password. (In case you want to use a
```

Comment:
What is wrong with the empty password?

Reply:
As far as I am aware it is not possible with current versions MySQL versions to create a database without a password.
At least I was not able to do so via the _MySQL Workbench_ on Windows and the shell on my Ubuntu distribution.

If I did not make a mistake there, this would make the empty password description obsolete and describing more closely how to set the password may be beneficial.

Reply:
At least for me, an empty password was not a problem when using the provided mysql.yml docker-compose file.
This is something you also removed from the documentation with this change and should definitely undo, as this is the easiest way to set up a local MySQL server and should therefore be mentioned here.

Reply:
Mmh, it might make a difference when using the docker container - I will revert the changes then and just add the second file that needs to be set in case of changing the password. Thanks for letting me know!

Reply:
Resolved in https://github.com/ls1intum/Artemis/pull/4276/commits/004608ec57ce99e5c16c4685e28707a608f11492